cumulusci.yml

```
@@ -140,6 +140,14 @@ tasks:
             path: unpackaged/config/wave
             package_xml: manifests/wave.xml
 
+    create_users:
```

@skristem maybe could be a bit more explicit as to what kind of users it's creating

```suggestion
    create_community_users:
```